### DISCUSSION TITLE
Add button "return to the CP" on the Entry listing page if user doesn't have Edit entry permission

### DISCUSSION THREAD

**Discussion by Yulyaswan** [ExpressionEngine/ExpressionEngine#3536] [Feature Requests]
### Suggest an idea for ExpressionEngine

Original PR #3196 
If a user has  the "Create entries" permission but not the "Edit own entries" permission, I get the `Error: You are not authorized to access this page error message.` 

I think we need to add something understandable to return him to the main CP.
My first thought was "Oh no, what I have already broken..." 
It can be a regular sidebar, and show the error only on the right side, instead of an Entry list or maybe the button "Return to the main CP"

And the second item here. If I have permission to `Create entries `only after saving this entry I got the same `Error: You are not authorized to access this page error message.` which is very confusing.

https://github.com/ExpressionEngine/ExpressionEngine/assets/23382425/877453a9-fe23-473e-8d75-7d92e7c9d811

**Comment by intoeetive** [comment]
Nice idea! See the implementation here: https://github.com/ExpressionEngine/ExpressionEngine/pull/3732
